extensions/wikia/ContributionPrototype/CPArticleRenderer.php

```diff
@@ -69,15 +69,19 @@ private function addStyles(OutputPage $output) {
 	}
 
 	private function addScripts(OutputPage $output) {
-		$output->addScript("<script src=\"{$this->publicHost}/public/assets/vendor.dll.js\"></script>");
-		$output->addScript("<script src=\"{$this->publicHost}/public/assets/app.js\"></script>");
+		$output->addHTML("<script src=\"{$this->publicHost}/public/assets/vendor.dll.js\"></script>");
+		$output->addHTML("<script src=\"{$this->publicHost}/public/assets/app.js\"></script>");
 	}
 
-	private function getArticleContent($title) {
+	private function getArticleContent($title, $action) {
 //		$internalHost = $this->urlProvider->getUrl(self::SERVICE_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)